.github/workflows/check.yml

```diff
@@ -0,0 +1,29 @@
+name: Gradle Check
+on: [pull_request]
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          java-version: 11
+          distribution: adopt
+      - name: Run Gradle
+        run: |
+          echo "./gradlew check --no-daemon --no-scan"
+          ./gradlew check --no-daemon --no-scan
```

Comment:
AFAIK if this is the last line the build fails and we're done, no?

Reply:
Let me know if you are suggesting the `run` something like:

```console
echo "./gradlew check --no-daemon --no-scan --parallel"
./gradlew check --no-daemon --no-scan ----parallel
GRADLE_CHECK_STATUS=$?

if [ "$GRADLE_CHECK_STATUS" != 0 ]
then
   echo Gradle Check Failed!
   exit $GRADLE_CHECK_STATUS
fi
          `